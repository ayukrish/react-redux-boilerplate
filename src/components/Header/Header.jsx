import React from 'react';
import styles from './Header.scss';

const Header = () => (
  <header className={styles.header}>
    header here
  </header>
);

export default Header;
