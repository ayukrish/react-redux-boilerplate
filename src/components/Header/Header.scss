@font-face {
  font-family: 'Antaro';
  src: url('../../assets/fonts/Antaro.ttf')  format('truetype'), /* Safari, Android, iOS */
}

.header {
  font-family: Antaro, sans-serif;
  color: red;
}