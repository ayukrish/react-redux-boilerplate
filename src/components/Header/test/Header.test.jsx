import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

// Components
import Header from '../Header';


describe('Header Test Suite', () => {
  it('renders correctly', () => {
    const output = shallow(
      <Header />,
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
