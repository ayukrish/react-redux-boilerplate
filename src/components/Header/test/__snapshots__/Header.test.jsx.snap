// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test Suite renders correctly 1`] = `
<header>
  header here
</header>
`;
